src/backend/gitporter/gitprovider/github.py

```diff
@@ -8,17 +8,35 @@
 import httpx
 from .common import RemoteRepositoryInformation
 
+class GithubRepoItemOwner(t.TypedDict):
+    login: str
+    avatar_url: str
 
-class GithubRepoListResponse(t.TypedDict):
-    name: str
+class GithubRepoItem(t.TypedDict):
+    name: str 
     full_name: str
-    git_url: str
-    ssh_url: str
     clone_url: str
+    owner: GithubRepoItemOwner
 
-def get_remote_repositories(workspace: str) -> t.List[RemoteRepositoryInformation]:
-    response = httpx.get(f"https://api.github.com/users/{workspace}/repos")
+def get_remote_repositories(access_token: str) -> t.List[RemoteRepositoryInformation]:
+    github_headers = {
+        "Accept": "application/vnd.github+json", 
+        "Authorization": f"Bearer {access_token}", 
```

Comment:
dafür hatte ich doch den HttpxBearerAuth in common.py

Reply:
Habe ich gesehen, aber ich fand es so ein bisschen simpler

Reply:
naja. erlaube ich